app/src/main/java/mod/pranav/dependency/resolver/DependencyResolver.kt

```diff
@@ -147,6 +147,8 @@ class DependencyResolver(
                 callback.onDependencyResolveFailed(e)
                 return@forEach
             }
+            callback.log("Cannot download ${artifact.toStr()}")
```

Comment:
is the artifact really not downloadable when it reaches that statement?

Reply:
I guess so, since the artifact failed earlier, it's possible that user configuration is incorrect or the system refused the request, or the error would have been logged and shown

Reply:
wdym failed earlier? what I meant is are you sure you added that log statement in the right place since it looks like that'll be called every time a valid dependency is resolved

Reply:
yes

Reply:
epic